src/box/gc.c

```diff
@@ -130,21 +146,31 @@ gc_init(on_garbage_collection_f on_garbage_collection)
 		panic("failed to start checkpoint daemon fiber");
 	fiber_set_joinable(gc.checkpoint_fiber, true);
 
+	gc.sync_fiber = fiber_new_system("gc_sync_daemon",
+					 gc_sync_fiber_f);
+	if (gc.sync_fiber == NULL)
+		panic("failed to start gc sync daemon fiber");
```

Comment:
Nit: No need to add "daemon" to the fiber name: `"gc_sync"`, `"failed to start gc sync fiber"`. Checkpoint daemon is named this way historically.

Reply:
Thanks, this "daemon" thing is a bit confusing. Removed it from the new fiber's name.